src/gap-skipper.js

```diff
@@ -17,6 +17,7 @@ export default class GapSkipper {
   /**
   * Represents a GapSKipper object.
   * @constructor
+  * @param {object} the options object that includes the tech
```

Comment:
Should be `@param {object} options object that includes the tech and settings`

Specifically the parameter name comes after the type